### PR TITLE
Allow server package to compile on non-linux platforms

### PR DESCRIPTION
**Pull request by oxtoacart**

### CODE REVIEW

File: server/server_nonlinux.go [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+package server
+
+import "net"
+
+type server struct{}
+
+// NewServer constructs a new unstarted packetforward Server. The server can be started by
+// calling Serve(). On non-linux platforms, this server does nothing.
+func NewServer(opts *Opts) (Server, error) {
+	return &server{}, nil
```
Comment (hwh33) [inline server/server_nonlinux.go]: Perhaps all of these functions should return an error (e.g. `errors.New("unsupported OS")`).  What do you think?

**Comment by hwh33** [comment]
Sorry, I somehow missed this earlier!

File: server/server_nonlinux.go (hunk 1 above)
Comment (oxtoacart) [reply server/server_nonlinux.go]: Good idea, will do.

**Comment by oxtoacart** [comment]
@hwh33 Incorporated your suggestion.

**Comment by hwh33** [comment]
Looks good to me!  The build is failing, but that is surely not related to this PR.

**Review by hwh33 — APPROVED** [review]

**Comment by oxtoacart** [comment]
Thanks @hwh33!

**Pull request merged**